ubuntu.22.04/Dockerfile

```diff
@@ -70,7 +73,13 @@ RUN DOTNET_CLI_TELEMETRY_OPTOUT=1 && \
 RUN apt-get install -y openjdk-11-jdk-headless=${Java_Jdk_Version}
 
 # Install common Java tools
-RUN apt-get install -y maven gradle
+RUN apt-get update && \
+    apt-get install -y maven gradle
+
+# Get Libssl 1.1, required for Azure CLI as of Ubuntu 22.04. This can be removed when a proper Azure CLI ubuntu 22.04 version is avaliable

```

Comment:
The Azure CLI needs a proper Ubuntu 22.04 release available. Using the old tool requires LibSSL 1.1, this is intended to be removed when a valid version is released.